<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestão de Menus</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_opcional">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcional</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_adicional">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicional</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_maxsel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. Sel.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sequencia">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequência</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_novo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Novo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atualizar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancelar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget_menu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_apagar">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar
Seleção</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
